test/test_3d_cpu_vs_gpu.py

```diff
@@ -0,0 +1,175 @@
+import unittest
+import numpy as np
+import os
+import timeit
+from ccpi.filters.regularisers import (
+    ROF_TV,
+    FGP_TV,
+    PD_TV,
+    SB_TV,
+    TGV,
+    LLT_ROF,
+    FGP_dTV,
+    NDF,
+    Diff4th,
+)
+from ccpi.filters.utils import cilregcuda
+
+gpu_modules_available = cilregcuda is not None
```

Comment:
This may not be sufficient to run the GPU tests. 

In CIL we also [verify ](https://github.com/TomographicImaging/CIL/blob/b066d588dc6de0188fd33dafd8e5e669b7434b7e/Wrappers/Python/test/utils.py#L55-L62)that a NVIDIA GPU is installed by calling `nvidia-smi` along these lines

```python
#nvidia
try:
    subprocess.check_output('nvidia-smi')
    has_nvidia = True
except:
    has_nvidia = False
```

Reply:
sure, will add that. thanks.